ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/export-actions/export-actions.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import styled from '../../styles/styled-components'
+
+const Root = styled.div`
+  overflow: auto;
+
+  .export-action {
+    cursor: pointer;
+    display: block;
+    line-height: ${props => props.theme.minimumButtonSize};
+    padding: 0px 15px;
+    background-color: ${props => props.theme.backgroundDropdown};
+  }
+
+  .export-action:hover {
+    background-color: ${props => props.theme.backgroundAccentContent};
+  }
+`
+
+type Action = {
+  title: string
+  url: string
+}
+
+type Props = {
+  actions: Action[]
+}
+
+function alphabetizeExportActions(actions: Action[]) {
+  return actions.sort((action1: Action, action2: Action) => {
+    if (action1.title > action2.title) return 1
+    if (action1.title < action2.title) return -1
```

Comment:
✏️ You could consider making the strings lower case before performing the comparison